server/events/vcs/gitlab_client_test.go

```diff
@@ -27,6 +31,10 @@ func TestNewGitlabClient_BaseURL(t *testing.T) {
 			"https://custom.domain",
 			"https://custom.domain/api/v4/",
 		},
+		{
+			"https://custom.domain/",
+			"https://custom.domain/api/v4/",
+		},
```

Comment:
Can you add a test-case with a port as well `http://custom.domain:8080`.

Reply:
Can you also add a test-case with a base path: `https://custom.domain/basepath/`